airflow/config_templates/config.yml

```diff
@@ -1349,6 +1349,13 @@ operators:
 webserver:
   description: ~
   options:
+    access_denied_message:
+      description: |
+        The message displayed when an user tries to perform operations outside their rights.
```

Comment:
```suggestion
        The notification that appears when a user attempts to execute actions beyond their authorised privileges.
```

Reply:
I would like to keep it as message because this is not a notification. Maybe I can term it to warning instead

Reply:
okay so 
"The message displayed when a user attempts to execute actions beyond their authorised privileges."
?

Reply:
If not, the minimum suggestion is to correct `an user` to `a user` 🙂

Reply:
Yeah why not, thanks, making the changes